src/System.Management.Automation/engine/CommandCompletion/CommandCompletion.cs

```diff
@@ -625,7 +604,7 @@ private static bool NeedToInvokeLegacyTabExpansion(PowerShell powershell)
             return false;
         }
 
-        private static List<CompletionResult> InvokeLegacyTabExpansion(PowerShell powershell, string input, int cursorIndex, bool remoteToWin7, out int replacementIndex, out int replacementLength)
+        private static List<CompletionResult> InvokeLegacyTabExpansion(PowerShell powershell, string input, int cursorIndex, out int replacementIndex, out int replacementLength)
```

Comment:
This method should just be removed as it seems all calls are within this source file